static/docs/use-cases/shared-storage-on-nfs.md

```diff
@@ -0,0 +1,254 @@
+# Shared Storage on NFS
+
+Data is increasing exponentially in terms of size and information. In the modern
+software development environment, teams are working together on same dataset to
+get the results. It became necessary that data is accessible to everyone and
+every team member has a same updated dataset. For this example, we will be using
+NFS (Network File System) for storing and sharing files on the network.
+
+With DVC, we need not to copy the dataset on our local machine everytime when
+new data is added to dataset. We can set the `cache directory` on NFS server.
+The cached data will be present in the NFS server, which in turn will be fast to
+access and process requests faster.
+
+> With large data files it is better to set the cache directory to NFS. Not only
```

Comment:
It should be a regular paragraph that explains _why_ it's beneficial to use DVC in this case. Namely, for people their workspace looks the same way as they are alone, DVC "magically" manages links from the workspace to cache. People can work with multiple versions of a datasets without affecting each other and without creating duplicates (like a full copy of a folder with 1M images)